src/simple_logger.rs

```diff
@@ -0,0 +1,63 @@
+use crate::ios::ffi_for_sanity_tests::{LOG_SENDER, CoreLogMessageThreadSafe, CoreLogLevel};
+use log::{Level, Metadata, Record};
+use log::{LevelFilter, SetLoggerError};
+use chrono::Utc;
+
+static LOGGER: SimpleLogger = SimpleLogger;
```

Comment:
Is this needed? Seems that it's used only to initialize the logger.

Reply:
It is needed, but there is also an alternative: using Box(Log)
[Boxed logger](https://docs.rs/log/0.4.8/log/#use-with-std)
Do you prefer the Boxed approach?

Reply:
Ah, no, didn't see that in the doc.